libs/components/src/lib/select/select.spec.ts

```diff
@@ -0,0 +1,95 @@
+import {elementUpdated, fixture, getBaseElement} from '@vivid-nx/shared';
+import { Select } from './select';
+import '.';
+
+const COMPONENT_TAG = 'vwc-select';
+
+describe('vwc-select', () => {
+	let element: Select;
+
+	global.ResizeObserver = jest.fn()
+		.mockImplementation(() => ({
+			observe: jest.fn(),
+			unobserve: jest.fn(),
+			disconnect: jest.fn()
+		}));
+
+	beforeEach(async () => {
+		element = (await fixture(
+			`<${COMPONENT_TAG}></${COMPONENT_TAG}>`
+		)) as Select;
+	});
+
+	describe('basic', () => {
+		it('should be initialized as a vwc-select', async () => {
+			expect(element).toBeInstanceOf(Select);
+			expect(element.open).toEqual(false);
+			expect(element.disabled).toEqual(false);
+		});
+	});
+
+	describe('label', function () {
+		it('should set a select label if label is set', async function () {
+			const labelText = 'label';
+			element.label = labelText;
+			await elementUpdated(element);
+			const labelElement = element.shadowRoot?.querySelector('label');
+			expect(labelElement)
+				.toBeTruthy();
+			expect(labelElement?.textContent?.trim())
+				.toEqual(labelText);
+		});
+
+		it('should show label only if label is set', async function () {
+			const labelElement = element.shadowRoot?.querySelector('label');
+			expect(labelElement)
+				.toBeNull();
+		});
+	});
+
+	describe('disabled', function () {
+		it('should set disabled class for select  when disabled is true', async () => {
+			expect(element.shadowRoot?.querySelector('.disabled')).toBeFalsy();
+			element.toggleAttribute('disabled', true);
+			await elementUpdated(element);
+			expect(element.shadowRoot?.querySelector('.disabled')).toBeTruthy();
```

Comment:
Expectations in the end of the use-case:
```suggestion
      const disableClassExistsWithDisabledFalse = Boolean(element.shadowRoot?.querySelector('.disabled'));
      
			element.toggleAttribute('disabled', true);
			await elementUpdated(element);
			
			expect(disableClassExistsWithDisabledFalse).toBeFalsy();
			expect(element.shadowRoot?.querySelector('.disabled')).toBeTruthy();
```